release-notes/opendistro-for-elasticsearch-security.release-notes-1.13.0.0.md

```diff
@@ -17,6 +17,7 @@ Compatible with Elasticsearch 7.10.2
 * ConfigUpdateRequest should include only updated CType ([#953](https://github.com/opendistro-for-elasticsearch/security/pull/953))
 * Fix AuthCredentials equality ([#876](https://github.com/opendistro-for-elasticsearch/security/pull/876))
 * Revert "Using SAML subject_key and roles_key in the HTTPSamlAuthenticator ([#1019](https://github.com/opendistro-for-elasticsearch/security/pull/1019))
+* Fixed async search action names and system index ([#1033](https://github.com/opendistro-for-elasticsearch/security/pull/1033))
```

Comment:
IMO, that is not a bug in the security plugin. It is maintenance.

Reply:
@vrozov Updated.